_pages/mobile-interview-project.md

```diff
@@ -94,9 +94,9 @@ There is no pagination API - the endpoint returns the full list of employees. Th
 
 ## Use What You Know
 
-We’re looking to see you at your best – as such, please use the language and frameworks you’re most comfortable with on your chosen mobile platform. This means that if you’re most proficient with Objective-C or Java, use that – don’t feel pressured to use Swift or Kotlin!
-
-Similarly, don’t feel pressured to use SwiftUI, Compose, and/or other libraries and techniques that you are unfamiliar with. We’re not going to prefer answers in one language over another, but we will expect you to be comfortable and proficient with your choices.
+We’re looking to see you at your best – as such, please use the language and frameworks you’re most comfortable with on your chosen mobile platform. 
+This means that if you’re most proficient with Objective-C or Java, use that – while Swift or Kotlin are preferred, being comfortable and proficient with your choices is more imporant in our evaluation. 
+Similarly, please don't feel pressured to use SwiftUI or Compose.
```

Comment:
This ended up cutting out some words `and/or other libraries and techniques you're unfamiliar with`

Reply:
Whoops, thanks for catching that!  I reworded to `and/or other libraries and techniques with which you're unfamiliar` to avoid ending the sentence with a preposition.  Not sure if that is still frowned upon these days, but I thought it couldn't hurt to rearrange it.